workers/unity/Packages/com.improbable.gdk.core/Systems/CleanTemporaryComponentsSystem.cs

```diff
@@ -23,30 +24,32 @@ protected override void OnCreateManager()
             foreach (var type in ReflectionUtility.GetNonAbstractTypes(typeof(IComponentData),
                 typeof(RemoveAtEndOfTickAttribute)))
             {
-                componentGroupsToRemove.Add((GetComponentGroup(ComponentType.ReadOnly(type)), type));
+                componentGroupsToRemove.Add((GetEntityQuery(ComponentType.ReadOnly(type)), type));
             }
 
             foreach (var type in ReflectionUtility.GetNonAbstractTypes(typeof(ISharedComponentData),
                 typeof(RemoveAtEndOfTickAttribute)))
             {
-                componentGroupsToRemove.Add((GetComponentGroup(ComponentType.ReadOnly(type)), type));
+                componentGroupsToRemove.Add((GetEntityQuery(ComponentType.ReadOnly(type)), type));
             }
         }
 
         protected override void OnUpdate()
         {
             Profiler.BeginSample("RemoveRemoveAtEndOfTick");
-            foreach ((var componentGroup, var componentType) in componentGroupsToRemove)
+            foreach ((EntityQuery componentGroup, var componentType) in componentGroupsToRemove)
```

Comment:
Nit: class & local variables should be renamed too (`componentGroup` -> `entityQuery`)

Also mixing types and var in the tuple deconstruction reads a little weird

Reply:
Not sure about the variable renaming. Unity's examples haven't changed, including the internal tests. I'm up for changing this, but maybe after this PR?

Reply:
If we don't do it here, it will get forgotten about 😉